datahub-web-react/src/app/home/HomePagePosts.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Divider, Typography } from 'antd';
+import styled from 'styled-components';
+import { useListPostsQuery } from '../../graphql/post.generated';
+import { Post, PostContentType } from '../../types.generated';
+import { PostTextCard } from '../search/PostTextCard';
+import { PostLinkCard } from '../search/PostLinkCard';
+
+const RecommendationContainer = styled.div`
+    margin-bottom: 32px;
+    max-width: 1000px;
+    min-width: 750px;
+`;
+
+const RecommendationTitle = styled(Typography.Title)`
+    margin-top: 0px;
+    margin-bottom: 0px;
+    padding: 0px;
+`;
+
+const ThinDivider = styled(Divider)`
+    margin-top: 12px;
+    margin-bottom: 12px;
+`;
+
+const LinkPostsContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+export const HomePagePosts = () => {
+    const { data: postsData } = useListPostsQuery({
```

Comment:
Does this endpoint filter by created time? It should in case > 10 posts are created

Reply:
It does now :)